src/issue-filing/common/http-query-builder.ts

```diff
@@ -43,6 +43,8 @@ export class HTTPQueryBuilder {
 
         const fullUrl = `${this.baseUrl}?${queryParameters}`;
 
+        // The following warning is thrown incorrectly because the call to 'truncate' is not on the file system
```

Comment:
Would be good to link the bug tracking the false positive, https://github.com/nodesecurity/eslint-plugin-security/issues/54